pkg/provisioner/block/block.go

```diff
@@ -114,9 +114,13 @@ func (block *blockProvisioner) Provision(options controller.VolumeOptions, ad *i
 	//volumeName := mapVolumeIDToName(*newVolume.Id)
 	filesystemType := resolveFSType(options)
 
-	metadata, err := block.metadata.Get()
-	if err != nil {
-		return nil, err
+	region, ok := os.LookupEnv("OCI_VOLUME_PROVISIONER_REGION")
```

Comment:
is there a more common env var that should be used? For example, when you look at AWS you always use AWS_REGION?

Reply:
Good idea. `OCI_REGION` then (though I'd expect that to be the long region name generally...)?

Reply:
@prydie my thoughts exactly. Seems there needs to be a better distinction between the canonical region and the region (short) across projects.